File: phaseGateOne/javaFiles/IncreasingOrder.java
import java.util.Scanner;

public class IncreasingOrder{

	public static void main(String... phasegate){

		Scanner input = new Scanner (System.in);
		
		System.out.print("\nEnter first number: ");
		int numberOne = input.nextInt();

		System.out.print("\nEnter second number: ");
		int numberTwo = input.nextInt();

		System.out.print("\nEnter third number: ");
		int numberThree = input.nextInt();
		

		

		if (numberTwo > numberOne && numberTwo > numberThree) {
			if (numberOne > numberThree) System.out.print("The numbers in increasing order " + numberThree + "," + numberOne + "," + numberTwo);
			else System.out.print("The numbers in increasing order " + numberOne + "," + numberThree  + "," + numberTwo);

			}

		if (numberOne > numberTwo && numberOne > numberThree) {
			if (numberTwo > numberThree) System.out.print("The numbers in increasing order " + "," + numberThree + numberTwo + "," + numberOne);
			else System.out.print("The numbers in increasing order " + numberTwo + "," + numberThree  + "," + numberOne);

			}

		if (numberThree > numberTwo && numberOne < numberThree) {
			if (numberTwo > numberOne) System.out.print("The numbers in increasing order " + numberOne + "," + numberTwo + "," + numberThree);
			else System.out.print("The numbers in increasing order " + numberTwo + "," + numberOne  + "," + numberThree);

			}


	}
}

File: phaseGateOne/increasingorder.py
first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))
third_number = int(input("Enter third number: "))

if first_number > second_number and first_number > third_number :
	if second_number > third_number: 
		print (f"The numbers in increasing order are {third_number}, {second_number}, { first_number}")
	else:
		print (f"The numbers in increasing order are {second_number}, {third_number}, { first_number}")

	
if second_number > third_number and second_number > first_number :
	if first_number > third_number: 
		print (f"The numbers in increasing order are {third_number}, {first_number}, { second_number}")
	else: 
		print (f"The numbers in increasing order are {first_number}, {third_number}, {second_number}")

if third_number > first_number and third_number > second_number :
	if first_number >  second_number: 
		print (f"The numbers in increasing order are {second_number}, {first_number}, {third_number}")
	else: 
		print (f"The numbers in increasing order are {first_number}, {second_number}, {third_number}")

